### DISCUSSION TITLE
keeps crashing, needs help.   see the screenshot for details. using win 10 professional, 8g RAM

### DISCUSSION THREAD

**Discussion by WangYunchao2023** [VioletteChiara/AnimalTA#13] [Q&A]
![image](https://github.com/VioletteChiara/AnimalTA/assets/131518923/97ed9aa5-a013-4906-8f9a-749bca4e75d0)

**Comment by VioletteChiara** [comment]
Hello, 
Could you please tell me at which moment this happen? Is it when you try to load the video for the first time?

**Reply by WangYunchao2023** [reply]
thanks for reply.
when trys to analyze at any steps if the video reader is opened, sometime crashes, some times doesnt.  
i 've changed to another computer, it s more stable now.
